src/NewsService.Business/EditNewsCommand.cs

```diff
@@ -41,6 +41,18 @@ public class EditNewsCommand : IEditNewsCommand
 
     public OperationResultResponse<bool> Execute(Guid newsId, JsonPatchDocument<EditNewsRequest> request)
     {
+      DbNews dbNews = _repository.Get(newsId);
```

Comment:
For what for dou you pull news in command? no need to pull the news, only check id

Reply:
you can do it before start edit in Edit method of repo

Reply:
done